HIPs/hedera-hashgraph.md

```diff
@@ -0,0 +1,304 @@
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the JavaScript SDK that today can be found at https://github.com/hashgraph/hedera-sdk-js
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Swift SDK that today can be found at https://github.com/hashgraph/hedera-sdk-swift
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Rust SDK that today can be found at https://github.com/hashgraph/hedera-sdk-rust
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Java SDK that today can be found at https://github.com/hashgraph/hedera-sdk-java
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Go SDK that today can be found at
+  https://github.com/hashgraph/hedera-sdk-go
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the C++ SDK that today can be found at https://github.com/hashgraph/hedera-sdk-cpp
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Solo CLI that today can be found at
+  https://github.com/hashgraph/solo
+
+As shown, the Hedera ecosystem already contains several sub-projects that must be part of `PROJECT`.
+All those sub-projects are necessary to deploy and run a full Hashgraph-based network and interact with that network.
+All mentioned projects are currently part of the Hashgraph organization at GitHub.
+Supplemental repositories include supporting tooling to deploy and validate a network built using the above codebases.
+In addition to the aforementioned repositories and projects, the community and ecosystem around Hedera and its unique
+hashgraph consensus algorithm is quite extensive, and we will want to transition a number of third-party projects
+coming from the community as an initial step once `PROJECT` has been set up. 
+An example of such a community project is https://github.com/OpenElements/hedera-enterprise.
+
+## GitHub issues
+The Hedera projects use GitHub issues today.
+Together, the projects contain more than 1,000 open issues that contain information that should be preserved since they
+contain great contributions from the community.
+By transferring the repositories to the new GitHub organization all issues will be transferred, too.
+
+## GitHub Actions 
+Next to the issues, each of the repositories uses GitHub Actions for continuous integration.
+While the GitHub Action workflow definitions are stored as part of the repository, we use custom action runners on
+hardware in the Google Cloud.
+The default GitHub action runners can not be used due to the complexity of `PROJECT`, especially for the services sub-project.
+To continue the development of `PROJECT` based on best practices and continuous integration, we propose to set up
+clones of the current hardware and action runners.
+The Hedera Hashgraph projects define teams with specific authorities.
+Those configurations should be migrated, and maintainers and committers for each sub-project should be defined.
```

Comment:
Thank you, please also work towards a process for maintainer's journey within the project.

Reply:
@arsulegai I already updated the text regarding the action runners. Can you please check if the new version fits you?

Reply:
> a process for maintainer's journey
What do you mean with that comment?

Reply:
@hendrikebbers , I was referring to setting up a document for new contributors. They will use this document to learn how can they become one the maintainers someday with their persistent contribution efforts.

Reply:
> @arsulegai I already updated the text regarding the action runners. Can you please check if the new version fits you?

Thank you for clarifying that Hedera will continue to provide these runners on Google Cloud. It would be helpful to state the reason for this requirement either in a project's improvement plan or as an issue within the project's repositories. Future developers or contributors can reference the document to understand and eventually remove these runners' dependency.